fastlane/metadata/fr-FR/promotional_text.txt

```diff
@@ -1 +1 @@
-Gérez votre boutique depuis n’importe où
+Gérez votre boutique depuis n’importe où
```

Comment:
This confirms we have some glitches in our tools around newlines being different, either depending on the machine we're running them on, or depending on where we pull the data from (ASC when we re-sync'd the values, or GP when we pull translations)… Gonna have to look at that at some point